iris-mpc-upgrade/Cargo.toml

```diff
@@ -49,3 +61,7 @@ path = "src/bin/tcp_ssl_upgrade_client.rs"
 [[bin]]
 name = "seed-v1-dbs"
 path = "src/bin/seed_v1_dbs.rs"
+
+[[bin]]
+name = "seed-v2-dbs"
```

Comment:
nit: probably we could use a better name here, matching the protocol instead of v1,v2

Reply:
This really is a generic seed binary that produces valid, like-production DBs for SMPCv2.